testing/kuttl/e2e/delete/13-delete-cluster-and-check.yaml

```diff
@@ -25,16 +25,20 @@ commands:
       if [ -z "$REPLICA" ]; then exit 1; fi
 
       kubectl delete postgrescluster -n "${NAMESPACE}" delete-switchover
+      
+      kubectl wait pod/"${REPLICA}" --namespace "${NAMESPACE}" --for=delete --timeout=-1s
```

Comment:
❓ `--timeout=-1s` means one week. Can we do anything sooner, like `5m` or `1h`?

🤔 How does this interact with the KUTTL timeout (~3m iirc?)

Reply:
Just ran a test and a sleep greater than the timeout does cause the test to error with a timeout. So no matter what we set this timeout to, if it's over 180 seconds, KUTTL will fail.